src/test/util_tests.cpp

```diff
@@ -170,6 +168,10 @@ BOOST_AUTO_TEST_CASE(span_write_bytes)
     mut_bytes[1] = std::byte{0x11};
     BOOST_CHECK_EQUAL(mut_arr.at(0), 0xaa);
     BOOST_CHECK_EQUAL(mut_arr.at(1), 0x11);
+    BOOST_CHECK_EQUAL(
+        HexStr(Span{ParseHex_expected}.first(5)),
+        "04678afdb0"
+    );
```

Comment:
wrong place for this test?

Reply:
Do you plan to address this or leave it?

Reply:
Done.